### ISSUE TITLE
[Survival Beta] F5 Clipping

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#5474]
The player can still go into no-clip mode in survival

**Comment by SovietSponge** [comment]
That could still be useful to have. Maybe just have the option to turn it on only when the "you're stuck" message appears and then turn it off after like 5 seconds?

**Comment by Xerma** [comment]
Also have to consider multiplayer where it could be heavily abused

**Comment by tsunamayo** [comment]
@Xerma Yes, I think we still need this for now but it is like cheating. Same think with F6, could be used to scout for mineral for example.
